src/outline.rs

```diff
@@ -426,13 +426,17 @@ impl SegmentSet {
                 ));
             }
         }
+
         // Finally, pop everything from the stack onto the tree
+        let mut last_sibling_idx = 0;
```

Comment:
@tirix  What is this for? It seems like only the `replace` should be needed for the fix: the `&sibling_stack[..=last_sibling_idx]` should be everything on the stack.

Reply:
You're right, that should be everything remaining on the stack.